docs/UserGuide.adoc

```diff
@@ -54,27 +54,35 @@ Not sure how to install Java? Visit the oracle website link below for more infor
 .  Download the latest `contactHeRo.jar` link:{repoURL}/releases[here].
 .  Copy the file to the folder you want to use as the home folder for contactHeRo, i.e. where  you want to store the `contactHeRo.jar` and the subsequently generated data files.
 .  Double-click the file to start the app.
+
 .  The contactHeRo welcome window should appear in a few seconds as shown below (Refer to figure 1).
 +
 .contactHeRo UI on launch
 image::UiOnStartup.png[width="800", align="center"]
 
 .  Login using the default username,`Admin` and default password, `ad123`. You should see the user interface as shown below (Refer to figure 2) in a few seconds.
```

Comment:
Change to:
Login using the default username,`Admin` and default password, `ad123` as shown below (Refer to figure 2).

Remove the few seconds part.